lib/atlas/dataset/derived_dataset.rb

```diff
@@ -0,0 +1,27 @@
+module Atlas
+  class Dataset::DerivedDataset < Dataset
+    GRAPH     = 'graph.yml'.freeze
+
+    attribute :base_dataset,      String
+    attribute :scaling,           Hash
+
+    validate :base_dataset_exists
+
+    def graph
+      YAML.load_file(File.join(directory, area, GRAPH))
+    end
+
+    # Overwrite
+    def dataset_dir
+      @dataset_dir ||= Atlas.data_dir.join(DIRECTORY).join(Dataset::FullDataset.find(base_dataset).key.to_s)
+    end
+
+    #######
+    private
+    #######
+
+    def base_dataset_exists
```

Comment:
I will do this often again with the missing parentheses - I am just to used to not liking them in simple cases :wink:
Thx for the reminder